src/main/scala-2.11/com/linagora/gatling/imap/action/ImapActionBuilder.scala

```diff
@@ -92,6 +100,24 @@ case class ImapFetchActionBuilder(requestName: String, sequence: Expression[Seq[
   override val actionName: String = "fetch-action"
 }
 
+case class ImapStoreActionBuilder(requestName: String, sequence: Expression[Seq[MessageRange]], flags: Expression[StoreFlags], private val checks: Seq[ImapCheck]) extends ImapCommandActionBuilder {
+  def check(checks: ImapCheck*): ImapStoreActionBuilder = copy(checks = this.checks ++ checks)
+
+  override def props(ctx: ImapActionContext): Props =
+    StoreAction.props(ctx, requestName, checks, sequence, flags)
+
+  override val actionName: String = "store-action"
+}
+
+case class ImapExpungeActionBuilder(requestName: String, private val checks: Seq[ImapCheck]) extends ImapCommandActionBuilder {
+  def check(checks: ImapCheck*): ImapExpungeActionBuilder = copy(checks = this.checks ++ checks)
+
+  override def props(ctx: ImapActionContext): Props =
+    ExpungeAction.props(ctx, requestName, checks)
+
+  override val actionName: String = "expunge-action"
```

Comment:
I know that is ugly but I had to return something of type `Validation[_]`, I could not found something better.
Any suggestion ?